extensions-contrib/kubernetes-overlord-extensions/src/main/java/org/apache/druid/k8s/overlord/KubernetesAndWorkerTaskRunner.java

```diff
@@ -268,6 +268,12 @@ public int getTotalCapacity()
     return Math.max(0, k8sCapacity) + Math.max(0, workerCapacity);
   }
 
+  @Override
+  public int getMaximumCapacity()
+  {
+    return workerTaskRunner.getMaximumCapacity() + kubernetesTaskRunner.getMaximumCapacity();
```

Comment:
The response for max capacity should depend on the underlying runner strategy.

Reply:
i can update this response in a separate PR, i think its not so simple to calculate what the capacity should be